tests/unit/multi.tcl

```diff
@@ -143,9 +143,9 @@ start_server {tags {"multi"}} {
 
         r multi
         r incr key
-        r exec
+        assert_equal [r exec] {}
         r debug set-active-expire 1
-    } {}
+    } {OK}
```

Comment:
we don't have to test the result of the debug command, so this could have just ended with:
```diff
-    } {OK}
+    }
```

but as i mentioned in my previous comment, since this test uses a debug command, which isn't always available on external test (i.e. when running the test suite against a redis compatible proxy, or someone that blocked the DEBUG command, we wanna skip it.
so we need to add a skip tag:

```suggestion
    } {OK} {needs:debug}
```